### ISSUE TITLE
Click point has to be too precise

### ISSUE THREAD

**Issue by kereid** [aodn/aodn-portal#2053]
Steps to reproduce:

Add a soop collection or argo.

While zoomed out click on the area where there might be a cluster of floats or a coupe of soop tracks

Zoom all the way in and click a track or a float

What should happen:

When there is a click in a general area of a float or track get some information

What does happen:

Have to be ridiculously precise to get any detail

When it's zoomed out is there someway to set the click area to encompass a larger surface? Even if this means getting details in the GFI for multiple floats or tracks it's better than the user clicking and thinking there is not information. 

Same for zooming in maybe the floats and tracks can be displayed bigger not still so small
![click presicion](https://cloud.githubusercontent.com/assets/2906602/10985197/5e1c5588-8473-11e5-8f16-3f25612f8db4.jpg)

**Comment by pmbohm** [comment]
Seeing errors that are the result of the portal not sending the correct click point when clicking over the date line. These requests will all silently fail. This may be responsible for a lot of misses
```
org.geoserver.platform.ServiceException: -965, 122 not in dimensions of image: 1535, 436
	at org.geoserver.wms.featureinfo.GetFeatureInfoKvpReader.read(GetFeatureInfoKvpReader.java:181)
```

**Comment by pmbohm** [comment]
Tested with styling changes for Argo and this seems to fix the issue
![screen shot 2015-11-23 at 5 31 07 pm](https://cloud.githubusercontent.com/assets/1066981/11331145/c22f6502-920a-11e5-94d4-e1c08c0d3a64.png)
Content issue created https://github.com/aodn/content/issues/154

**Comment by kereid** [comment]
![click point](https://cloud.githubusercontent.com/assets/2906602/11886457/6252ba9e-a57e-11e5-8927-3cdab148b435.jpg
Re-opening as it still seems to be hard to click on a track it's just specific to one or 2 collections.

**Comment by pmbohm** [comment]
Yes this specific to one or two collections. The bug needs to be reported for the collection on the content repo and the fix applied as described at aodn/content#154
